custom_components/kumo/manifest.json

```diff
@@ -7,8 +7,8 @@
     "dependencies": [],
     "codeowners": [ "@dlarrick" ],
     "requirements": ["pykumo==0.3.9"],
-    "version": "0.3.12",
-    "homeassistant": "2021.12.0",
+    "version": "0.3.13",
+    "homeassistant": "2024.1.0",
```

Comment:
I'm not actually sure what version we're compatible with, but I don't test with anything but the latest so I just picked something recent.